jsf-showcase-webapp/src/test/java/com/liferay/faces/test/showcase/multimediaimage/ImageGeneralTester.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2000-2016 Liferay, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.liferay.faces.test.showcase.multimediaimage;
+
+import org.junit.Test;
+
+import com.liferay.faces.test.selenium.Browser;
+import com.liferay.faces.test.selenium.assertion.SeleniumAssert;
+import com.liferay.faces.test.showcase.output.OutputTester;
+
+
+/**
+ * @author  Kyle Stiemann
+ * @author  Philip White
+ */
+public class ImageGeneralTester extends OutputTester {
+
+	@Test
+	public void runImageGeneralTest() throws Exception {
+
+		Browser browser = Browser.getInstance();
+		navigateToUseCase(browser, "graphicimage", "general");
+
+		// Test that the images render on the page successfully.
+		SeleniumAssert.assertElementVisible(browser,
+			"//div//img[contains(@src,'javax.faces.resource') and contains(@src,'ln=images')]/ancestor::div//label//a[contains(text(),'value')]");
```

Comment:
Please ensure that the test asserts that the `<img>` is visible. Currently, the test asserts that an `<a>` is visible. You can change this by ensuring that the `img` element is at the end of the xpath.